fastchat/protocol/chat_completion.py

```diff
@@ -14,6 +14,7 @@ class ChatCompletionRequest(BaseModel):
     n: int = 1
     max_tokens: Optional[int] = None
     stop: Optional[str] = None
+    seed: Optional[int] = 0
```

Comment:
Set the default value to `None` instead 0.  
The default behavior should not be deterministic.